src/contour/display/ShaderConfig.cpp

```diff
@@ -135,7 +137,7 @@ std::unique_ptr<QOpenGLShaderProgram> createShader(ShaderConfig const& _shaderCo
     }
 
     if (auto const logString = shader->log().toStdString(); !logString.empty())
-        errorlog()("Shader log: {}", logString);
+        ShaderLog()("Shader log: {}", logString);
```

Comment:
```suggestion
        ShaderLog()("{}", logString);
```

Because the logger will already prefix it with the shader log's tag. :)

Reply:
This may be worth mentioning in the `metainfo.xml`'s changelog section. What do you think?

Reply:
> Because the logger will already prefix it with the shader log's tag. :)

Oh, right, didn't notice